content/2021/08/simplex-uses-minfraud-data.mdx

```diff
@@ -1,29 +1,64 @@
+### “In the crypto market, everything changes quickly. Without an adaptive risk
+model, we would have to rely on a strict rule-based system. We might use simple
+IP geolocation and look at the distances involved in the transaction, or rely
+solely on one risk score. If we had to update these rules manually in response
+to emerging trends, our workflow would be very slow and inefficient.”
 
-For the analysts who work on Simplex’s fraud detection algorithm, data is key to a responsive algorithm. They decided to implement minFraud because, in addition to a risk score, the [minFraud Insights service](https://www.maxmind.com/en/solutions/minfraud-services) gave them access to a breadth of high quality data on [inputs ranging from IP geolocation and email risk to financial institution data](https://www.maxmind.com/en/solutions/minfraud-services/data-points#outputs), all via a flexible easy-to-use API.
+For the analysts who work on Simplex’s fraud detection algorithm, data is key to
+a responsive algorithm. They decided to implement minFraud because, in addition
+to a risk score, the [minFraud Insights
+service](https://www.maxmind.com/en/solutions/minfraud-services) gave them
+access to a breadth of high quality data on [inputs ranging from IP geolocation
+and email risk to financial institution
+data](https://www.maxmind.com/en/solutions/minfraud-services/data-points#outputs),
+all via a flexible easy-to-use API.
 
-### “The more data I have, the more significant my results can be. More data means that we can do better research.”
+### “The more data I have, the more significant my results can be. More data
+means that we can do better research.”
 
-The variety of data available through minFraud Insights allows Simplex to track patterns across many different data points. Rather than trying to anticipate which data points will be most relevant to fraud, Simplex can feed the breadth of minFraud data into their algorithm, allowing it to identify and adapt to patterns in fraud as they emerge. This adaptability is key to fighting fraud in the rapidly changing crypto industry.
+The variety of data available through minFraud Insights allows Simplex to track
+patterns across many different data points. Rather than trying to anticipate
+which data points will be most relevant to fraud, Simplex can feed the breadth
+of minFraud data into their algorithm, allowing it to identify and adapt to
+patterns in fraud as they emerge. This adaptability is key to fighting fraud in
+the rapidly changing crypto industry.
 
-### “We also make minFraud data available to our fraud analysts for manual review of transactions.”
+### “We also make minFraud data available to our fraud analysts for manual
```

Comment:
I think this should be a block quote rather than a header. However, if it is preferred to stay a header, only part of the quote is currently a heading.